Update Close Example(s) (#86)
LivePersonInc/node-agent-sdk: README.md

`"OK Agent added successfully"`

##### Example: Close a conversation
This will immediately close the conversation and any associated dialogs.

> >Note: If the account is configured for post-conversation survey (PCS), the survey dialog will not be triggered. To allow PCS, without closing the conversation, the conversation's main dialog can be closed (see the next example).

Reviewer: ```suggestion
>Note: If the account is configured for post-conversation survey (PCS), the survey dialog will not be triggered. To allow PCS without closing the conversation, the conversation's main dialog should be closed instead (see the "Close Dialog" example).
```
